CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Features**:
+
+- Introduce reservoir sampling rule. ([#2550](https://github.com/getsentry/relay/pull/2550))
```

Comment:
Let's move this to "Internal" since the user-visible feature merely indirectly makes use of this.